tests/CppUTest/UtestTest.cpp

```diff
@@ -225,7 +225,7 @@ TEST(UtestShell, TestStopsAfterSetupFailure)
 #ifdef NEEDS_DISABLE_UNREACHABLE_CODE
 # pragma GCC diagnostic push
 # pragma GCC diagnostic ignored "-Wunreachable-code"
-#endif /* NEEDS_DISABLE_UNREACHABLE_CODE */
```

Comment:
What about the if to make it reachable ?

Reply:
*(as above)*